compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -111,6 +114,13 @@ def add_feature_list(seq_example: tf.train.SequenceExample,
     add_function(seq_example, feature, feature_name)
 
 
+@dataclasses.dataclass
+class SequenceExampleFeatureNames:
```

Comment:
nit: add a comment or doc string saying what this is for ("feature names that we depend on" for example).

also use these in the test

Reply:
done.